src/variables.jl

```diff
@@ -27,12 +34,18 @@ function JuMP.build_variable(
         _error("Invalid fix value, must be false or true.")
     elseif info.has_start && !isone(info.start) && !iszero(info.start)
         _error("Invalid start value, must be false or true.")
+    elseif (info.has_start || info.has_fix) && !isnothing(logical_compliment)
+        _error("Cannot fix or provide a start value for logical variable " *
+               "that has a logical compliment variable.")
+    elseif !isnothing(logical_compliment) && has_logical_compliment(logical_compliment)
+        _error("Cannot specify a logical compliment that itself is a " *
```

Comment:
Maybe add something about using the logical_complement flag on only one of the variables in the disjunction